data/profiles/boot-livecd.ipxe

```diff
@@ -0,0 +1,4 @@
+kernel <%=repo%>/vmlinuz0 
+initrd <%=repo%>/initrd.img
+imgargs vmlinuz0 rootflags=loop initrd=initrd.img root=live:/<%=version%>.iso rootfstype=iso9660 !text !lang !ksdevice
```

Comment:
How should the user put the ISO image? I didn't see the "root=live:/<%=version%>.iso" reference the <%=repo%>